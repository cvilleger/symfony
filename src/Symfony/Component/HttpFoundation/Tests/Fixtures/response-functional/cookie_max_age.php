<?php

use Symfony\Component\HttpFoundation\Cookie;

$r = require __DIR__ . '/common.inc';

$r->headers->setCookie(new Cookie('foo', 'bar', 946749600, '', null, false, false));
$r->sendHeaders();

setcookie('foo', 'bar', 946749600, '/');
