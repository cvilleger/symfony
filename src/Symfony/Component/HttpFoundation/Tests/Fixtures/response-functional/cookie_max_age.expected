
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
    [1] => Cache-Control: no-cache, private
    [2] => Date: Sat, 12 Nov 1955 20:04:00 GMT
    [3] => Set-Cookie: foo=bar; expires=Sat, 01-Jan-2000 18:00:00 GMT; Max-Age=0; path=/
    [4] => Set-Cookie: foo=bar; expires=Sat, 01-Jan-2000 18:00:00 GMT; Max-Age=0; path=/
)
shutdown
